fizz-cli/fizz_cli/utils.py

```diff
@@ -73,10 +73,7 @@ def replace_route(yaml_data, new_route):
     if (
         not yaml_data
         or "spec" not in yaml_data
-        or (
-            "ingressconfig" not in yaml_data["spec"]
-            and "relativeurl" not in yaml_data["spec"]
```

Comment:
Add BlackConnect plugin to your ide then reformat and push this file again. If you don't want to do that then don't format the entire file.